docs/debugging-tests.md

```diff
@@ -26,13 +26,15 @@ Unfortunately debugging tests in the browser via Node is not great, but they are
 
 ## Debugging in VisualStudioCode
 
-* Open your JavaScript application.
-* Add a `debugger;` statement into the file and line you want to trigger a breakpoint, add a red check on the line number.
+* Open your JavaScript project (either the folder, or go straight for the file you want to debug and open that).
```

Comment:
The first point was more to suggest opening the project in Visual Studio, rather than the file. They should open the project directory in Visual Studio before choosing the file.

Reply:
Also, for me, opening the file was the only thing that actually worked!